docs/guides/setting-url-wordpress-source-data..md

```diff
@@ -0,0 +1,114 @@
+
+`state.source.api` can be set manually but it can be computed from the value of `state.source.url` in most of the cases
+
+## Wordpress scenarios
+
+### A self-hosted WordPress site 
+
+Most of the Frontity projects will use a custom domain for a self-hosted WordPress site ([wp.org](http://wp.org/)) like for example `https://test.frontity.org/`
+
+The recommended way for setting this URL is through the `state.source,url`
+
+```javascript
+// frontity.settings.js
+export default {
+  packages: [
+    {
+      name: "@frontity/wp-source",
+      state: {
+        source: {
+          url: "https://test.frontity.org",
+        },
+      },
+    },
+  ],
+};
+```
+
+In this example the computed values would be:
+- `state.source.api`:  `https://test.frontity.org/wp-json` (value computed from `state.source.url`)
+- `state.wpSource.isWpCom`: `false` (value derived `state.source.api`)
+
+The same recommendation applies for custom domains used with a [Business wordpress.com plan](https://wordpress.com/support/business-plan/) 
+
+
+### A Free wordpress.com plan 
+
+Some other Frontity projects will use a free wordpress.com installation with URL's pointing to subdomains of wordpress.com like for example  `https://frontitytest.wordpress.com/` 
+
+The recommended way for setting this URL is also through the `state.source,url`
+
+```javascript
+// frontity.settings.js
+export default {
+  packages: [
+    {
+      name: "@frontity/wp-source",
+      state: {
+        source: {
+          url: "https://frontitytest.wordpress.com/",
+        },
+      },
+    },
+  ],
+};
+```
+
+In this example the computed values would be:
+- `state.source.api`:  `https://frontitytest.wordpress.com//wp-json` (value computed from `state.source.url`)
+- `state.wpSource.isWpCom`: `true` (value derived `state.source.api`)
+
+### A Personal or Premium wordpress.com plan 
+
+A less frequent use case is using a Personal or Premium wordpress.com. These plans allows you to use a custom domain but this a special case because the final REST API url is available through a special URL
+
+The recommended way for setting this URL is also through the `state.source,url` but specifying it is a Personal or Premium wordpress.com plan installation by setting `state.wpSource.isWpCom` to `true`
+
+```javascript
+// frontity.settings.js
+export default {
+  packages: [
+    {
+      name: "@frontity/wp-source",
+      state: {
+        source: {
+          url: "https://test-premium-plan.frontity.org",
+        },
+        wpSource: {
+          isWpCom: true
+        } 
+      },
+    },
+  ],
+};
+```
+
+In this example the computed values would be:
+- `state.source.api`: ` https://public-api.wordpress.com/wp/v2/sites/test-premium-plan.frontity.org` (value computed from `state.source.url` & `state.wpSource.isWpCom`)
+
+---
+
+All these scenarios and their different combinations depending on how you set the values are described in detail in the following table
```

Comment:
I would not include this table. It was made for internal purposes when we needed to work on the internal logic of these props. Also, it could be a bit misleading because nobody should use `state.source.api` anymore.

I think the written explanations are super clear so I would only leave those 🙂